BingWallpaper/MainForm.cs

```diff
@@ -89,5 +89,17 @@ private void titleLable_Click(object sender, EventArgs e)
             if (titleLable.Tag != null) UrlHelper.OpenLink((string)titleLable.Tag);
         }
 
+        private void preButton_Click(object sender, EventArgs e)
+        {
+            if(Constants.PictureIndex > 0)
+            {
+                Program.ReflashPicture(--Constants.PictureIndex);
```

Comment:
`PictureIndex` shouldn't be in `Constants` anymore because it's not a constant now.